regparser/tree/xml_parser/appendices.py

```diff
@@ -250,6 +256,9 @@ def end_group(self):
             self.nodes = []
 
     def process(self, appendix, part):
+        #TODO: currently this fails the appendix parser test
+        #TODO: there should be a flag to check what sort of appendix we have
+        #TODO: if we have an appendix with headings like "A-1" or not
```

Comment:
If this fails the appendix parser test, we should fix it or change the test to take into account the cases where it fails and shouldn't.